opm/simulators/linalg/findOverlapRowsAndColumns.hpp

```diff
@@ -110,6 +110,28 @@ namespace detail
             }
         }
     }
+    template <class Grid>
+    size_t numInteriorCells(const Grid& grid, bool ownerFirst)
+    {
+        size_t numInterior = 0;
+        if (!ownerFirst)
+            return grid.numCells();
+        const auto& gridView = grid.leafGridView();
+        auto elemIt = gridView.template begin<0>();
+        const auto& elemEndIt = gridView.template end<0>();
+
+        // loop over cells in mesh
+        for (; elemIt != elemEndIt; ++elemIt) {
+            const auto& elem = *elemIt;
```

Comment:
In general (not a CpGrid) this might cost something. Maybe prevent it?

Reply:
Just to be clear you men the line: (const auto& elem = *elemIt;)? 

It is now removed.